package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/embed",
-  "version": "2.4.1",
+  "version": "2.3.2",
   "keywords": [
```

Comment:
@Smokietr Please Change Accordingly.
Sorry for Earlier Typo which is `2.4.2`

Reply:
And Also Please update [service.md](https://github.com/editor-js/embed/blob/master/docs/services.md)